### ISSUE TITLE
OSX - libmkl_rt_path

### ISSUE THREAD

**Issue by ian-jeffrey** [KristofferC/Pardiso.jl#93]
Hello,

When running a new instance of Julia 1.9 and "using Pardiso" I get the following error:

```bash
_julia> using Pardiso
ERROR: InitError: UndefVarError: `libmkl_rt_path` not defined
Stacktrace:
 [1] __init__()
   @ Pardiso ~/.julia/packages/Pardiso/rrIt7/src/Pardiso.jl:113
 [2] register_restored_modules(sv::Core.SimpleVector, pkg::Base.PkgId, path::String)
   @ Base ./loading.jl:1115
 [3] _include_from_serialized(pkg::Base.PkgId, path::String, ocachepath::String, depmods::Vector{Any})
   @ Base ./loading.jl:1061
 [4] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String, build_id::UInt128)
   @ Base ./loading.jl:1506
 [5] _require(pkg::Base.PkgId, env::String)
   @ Base ./loading.jl:1783
 [6] _require_prelocked(uuidkey::Base.PkgId, env::String)
   @ Base ./loading.jl:1660
 [7] macro expansion
   @ ./loading.jl:1648 [inlined]
 [8] macro expansion
   @ ./lock.jl:267 [inlined]
 [9] require(into::Module, mod::Symbol)
   @ Base ./loading.jl:1611
during initialization of module Pardiso
```

When repeating the command I get:

```bash
julia> using Pardiso

julia> 
```
However, once I try to run the exampleherm.jl example, I get the following error:

```bash
X1 = solve(ps, A, b)
ERROR: could not load library ""
dlopen(.dylib, 0x0001): tried: '/Applications/Julia-1.9.app/Contents/Resources/julia/lib/julia/.dylib' (no such file), '/Applications/Julia-1.9.app/Contents/Resources/julia/lib/julia/../.dylib' (no such file), '/Applications/Julia-1.9.app/Contents/Resources/julia/bin/../lib/.dylib' (no such file), '.dylib' (no such file), '/usr/local/lib/.dylib' (no such file), '/usr/lib/.dylib' (no such file), '/Users/ianjeffrey/.dylib' (no such file)
Stacktrace:
 [1] ccall_pardisoinit(ps::MKLPardisoSolver)
   @ Pardiso ~/.julia/packages/Pardiso/rrIt7/src/mkl_pardiso.jl:49
 [2] pardisoinit
   @ ~/.julia/packages/Pardiso/rrIt7/src/Pardiso.jl:214 [inlined]
 [3] solve!(ps::MKLPardisoSolver, X::Vector{ComplexF64}, A::SparseMatrixCSC{ComplexF64, Int64}, B::Vector{ComplexF64}, T::Symbol)
   @ Pardiso ~/.julia/packages/Pardiso/rrIt7/src/Pardiso.jl:257
 [4] solve
   @ ~/.julia/packages/Pardiso/rrIt7/src/Pardiso.jl:222 [inlined]
 [5] solve(ps::MKLPardisoSolver, A::SparseMatrixCSC{ComplexF64, Int64}, B::Vector{ComplexF64})
   @ Pardiso ~/.julia/packages/Pardiso/rrIt7/src/Pardiso.jl:221
 [6] top-level scope
   @ REPL[15]:4
```

I can reproduce this issue on different machines (both macOS Monterey and Apple M1 chips) in both Julia 1.8 and 1.9. 

Any help/insight would be much appreciated - thank you in advance.

**Comment by mipals** [comment]
I am having the same issue. Did you by any change figure out how to solve this @ian-jeffrey ?

**Comment by ViralBShah** [comment]
Fixed in #95
